src/soc/ibm/power9/istep_13_8.c

```diff
@@ -596,16 +596,16 @@ static void thermal_throttle_scominit(uint8_t chip, int mcs_i, int mca_i)
 	    // Set to disable permanently due to hardware design bug (HW403028) that won't be changed
 	    [53]    MBA_FARB3Q_CFG_NM_CHANGE_AFTER_SYNC = 0
 	*/
-	printk(BIOS_EMERG, "Please FIXME: ATTR_MSS_RUNTIME_MEM_THROTTLED_N_COMMANDS_PER_SLOT\n");
-	/* Values dumped after Hostboot's calculations, may be different for other DIMMs */
+	/* Values come from talos.xml, may be different for other platforms */
```

Comment:
I would be more specific to simplify looking for origin of the values for the reader.
```suggestion
	/*
	 * Values of m_dram_clocks and nm_throttled_n_per_port come from talos.xml
	 * nm_n_per_slot and nm_n_per_port are derived from values in talos.xml
	 *
	 * All of them may be different for other platforms
	 */
```

Reply:
Done